net.sourceforge.projects.vht.yml

```diff
@@ -8,12 +8,6 @@ finish-args:
   - --filesystem=xdg-run/pipewire-0
   - --socket=wayland
   - --socket=fallback-x11
+  - --share=ipc
 modules:
-  - name: vht
-    buildsystem: simple
-    build-commands:
-      - pip3 install . --no-build-isolation --prefix=${FLATPAK_DEST}
-    sources:
-      - type: git
-        url: https://github.com/rdybka/vht
-        tag: v0.3.97
+  - python3-vht.json
```

Comment:
no how it was was fine.

I asked about the dependencies because upstream mention thee dependency `pipewire-audio-client-libraries [jack_capture]` and I didn't see it installed.

Reply:
oh, these are just to build it on a generic linux box

Reply:
jack_capture can get invoked for rendering of projects. The option will not display in sandbox as I'm not sure how to implement it yet.